src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,532 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses IntAct data and maps the interaction types to BEL."""
+
+from typing import Dict, Iterable, List
+from zipfile import ZipFile
+
+import pandas as pd
+from protmapper.uniprot_client import get_mnemonic
+from tqdm import tqdm
+
+import pybel.dsl
+from bio2bel.utils import ensure_path
+from pybel import BELGraph
+
+
+#: Relationship types in IntAct that map to BEL relation 'increases'
+INTACT_INCREASES_ACTIONS = {
+    'phosphorylation reaction',
+    'sumoylation reaction',
+    'methylation reaction',
+    'transglutamination reaction',
+    'ubiquitination reaction',
+    'acetylation reaction',
+    'adp ribosylation reaction',
+    'neddylation reaction',
+    'hydroxylation reaction',
+    'phosphotransfer reaction',
+    'glycosylation reaction',
+    'palmitoylation reaction',
+    'oxidoreductase activity electron transfer reaction',
+    'amidation reaction',
+    'dna strand elongation',
+    'isomerase reaction',
+    'isomerization  reaction',
+    'proline isomerization  reaction',
+    'sulfurtransfer reaction',
+    'deamination reaction',
+    'ampylation reaction',
+    'aminoacylation reaction',
+    'myristoylation reaction',
+    'lipid addition',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'decreases'
+INTACT_DECREASES_ACTIONS = {
+    # decreases
+    'deubiquitination reaction',
+    'protein cleavage',
+    'cleavage reaction',
+    'deacetylation reaction',
+    'lipoprotein cleavage reaction',
+    'dna cleavage',
+    'rna cleavage',
+    'dephosphorylation reaction',
+    'deformylation reaction',
+    'demethylation reaction',
+    'deneddylation reaction',
+    'lipid cleavage',
+    'deamidation reaction',
+    'decarboxylation reaction',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'association'
+INTACT_ASSOCIATION_ACTIONS = {
+    'physical association',
+    'association',
+    'colocalization',
+    'direct interaction',
```

Comment:
when you know that it's a directed interaction you can at least use "regulates" instead of "association" relation

Reply:
I identified it now as a 'regulates' interaction with the object_modifier being the activity. I found a GO term matching the activity

Reply:
Closed here: c5ea652cb742e2b7ca6c455484a1efd15015434d